app/uk/gov/hmrc/agentservicesaccount/models/SelectChanges.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.agentservicesaccount.models
+
+case class SelectChanges (businessName: Option[String], address: Option[String],  email: Option[String], telephone: Option[String]) {
+
+  def atLeastOneSelected: Boolean = businessName.nonEmpty || address.nonEmpty || email.nonEmpty || telephone.nonEmpty
+
+  def pagesSelected: Set[String] = Set(businessName, address, email, telephone).filter(_.nonEmpty).map(_.get)
+}
+
```

Comment:
adding this companion object will enable us to read/write the model from the repo, for example when mapping to a form from sessionCache
```suggestion
object SelectChanges {
  implicit val format: Format[SelectChanges] = Json.format[SelectChanges]
}
```